devel/include/client/current_time.h

```diff
@@ -0,0 +1,227 @@
+            }
+
+            static const char *value(const ::client::current_time_<ContainerAllocator> &) { return value(); }
+        };
+
+        template<class ContainerAllocator>
+        struct Definition<::client::current_time_<ContainerAllocator> > {
+            static const char *value() {
+                return "string name\n"
+                       "\n"
+                       "uint16 year\n"
+                       "uint16 month\n"
+                       "uint16 day\n"
+                       "\n"
+                       "uint16 hour\n"
+                       "uint16 minute\n"
+                       "uint16 second\n";
+            }
+
+            static const char *value(const ::client::current_time_<ContainerAllocator> &) { return value(); }
+        };
+
+    } // namespace message_traits
+} // namespace ros
+
+namespace ros {
+    namespace serialization {
+
+        template<class ContainerAllocator>
+        struct Serializer<::client::current_time_<ContainerAllocator> > {
+            template<typename Stream, typename T>
+            inline static void allInOne(Stream &stream, T m) {
+                stream.next(m.name);
+                stream.next(m.year);
+                stream.next(m.month);
+                stream.next(m.day);
+                stream.next(m.hour);
+                stream.next(m.minute);
+                stream.next(m.second);
+            }
+
+            ROS_DECLARE_ALLINONE_SERIALIZER
+        }; // struct current_time_
+
+    } // namespace serialization
+} // namespace ros
+
+namespace ros {
+    namespace message_operations {
+
+        template<class ContainerAllocator>
+        struct Printer<::client::current_time_<ContainerAllocator> > {
+            template<typename Stream>
+            static void
+            stream(Stream &s, const std::string &indent, const ::client::current_time_<ContainerAllocator> &v) {
+                s << indent << "name: ";
+                Printer<std::basic_string < char, std::char_traits < char>,
+                        typename ContainerAllocator::template rebind<char>::other > >
+                        ::stream(s, indent + "  ", v.name);
+                s << indent << "year: ";
+                Printer<uint16_t>::stream(s, indent + "  ", v.year);
+                s << indent << "month: ";
+                Printer<uint16_t>::stream(s, indent + "  ", v.month);
+                s << indent << "day: ";
+                Printer<uint16_t>::stream(s, indent + "  ", v.day);
+                s << indent << "hour: ";
+                Printer<uint16_t>::stream(s, indent + "  ", v.hour);
+                s << indent << "minute: ";
+                Printer<uint16_t>::stream(s, indent + "  ", v.minute);
+                s << indent << "second: ";
+                Printer<uint16_t>::stream(s, indent + "  ", v.second);
+            }
+        };
+
+    } // namespace message_operations
+} // namespace ros
+
+#endif // CLIENT_MESSAGE_CURRENT_TIME_H
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> #endif line should be "#endif  // DEVEL_INCLUDE_CLIENT_CURRENT_TIME_H_"  [build/header_guard] [5]